README.md

```diff
@@ -1,90 +1,111 @@
+* Wrappers for values derived from state with a callback function to modify them (`data.{ViewF, ViewOptF, ViewListF}`).
 
-Functional, tagless and lens-based, global state management. With `scalajs-react`, `fs2.Stream`, `monocle` integrations.
+Additionally, for `scalajs-react` apps it provides:
+* A root component to hold the application state (`AppRoot`).
+* A component that dynamically renders values from `fs2.Stream`s (`StreamRenderer`).
+* A component that dynamically renders values from `fs2.Stream`s, allowing children to modify the value too (`StreamRendererMod`).
+* Conversions between `Callback`s and `cats-effect` effects (`implicits._`).
+* `Reusability` and other utilities for `data.Pot` and `data.View*F` (`data.implicits._`)
 
-## Core
-
-### Model
-
-Model is just a `case class` with all the client state.
-
-`monocle` lenses are used to read/write to the model. `@Lenses` on Model are recommended.
-
-The model must be placed somewhere widely accessible. Eg: `object AppState`.
+The library takes a tagless approach based on `cats-effect`. Logging is performed via `log4cats`.
 
-It must be defined by calling `Model[F, M](<initial model>)`, where `F[_]` is the effect context to use and `M` is the model type.
-
-Eg, if you choose to use `IO`:
-```scala
-import scala.concurrent.ExecutionContext.global
-import crystal._
-import monocle.Lens
-import monocle.macros.Lenses
+## Core
 
-object AppState {
-  implicit private val timerIO: Timer[IO] = cats.effect.IO.timer(global)
-  implicit private val csIO: ContextShift[IO] = IO.contextShift(global)
+### Pot[A]
 
-  @Lenses
-  case class RootModel(
-                        todos: Pot[Todos],
-                        motd: Pot[String],
-                        motdInstant: Option[Instant],
-                        progress: Int
-                      )
+A `Pot[A]` represents a value of type `A` that has been requested somewhere and may or not be available yet, or the request may have failed.
 
-  val rootModel = Model[IO, RootModel](RootModel(Empty, Empty, None, 0))
-}
-``` 
+It is a sum type consisting of:
+- `Pending(<Long>)`, where the `Long` is meant to store the instant of creation, in millis since epoch. It is initialized to `System.currentTimeMillis()` by default.
+- `Ready(<A>)`.
+- `Error(<Throwable>)`.
 
-Note that the chosen effect context `F[_]` must have `cats.effects.{ConcurrentEffect, Timer}` type class instances.
+`Pot[A]` implements the following methods: 
+* `map[B](f: A => B): Pot[B]`
+* `fold[B](fp: Long => B, fe: Throwable => B, fr: A => B): B`
+* `flatten[B]: Pot[B]` (if `A <: Pot[B]`)
+* `flatMap[B](f: A => Pot[B]): Pot[B]`
+* `toOption: Option[A]`
+* `toTryOption: Option[Try[A]]`
 
-### FixedLens
+The `crystal.data.implicits._` import will provide:
+* Instances for `cats` `MonadError`, `Traverse`, `Align` and `Eq` (as long as there's an `Eq[A]` in scope).
+* Convenience methods: `<Any>.ready`, `<Option[A]>.toPot`, `<Try[A]>.toPot` and `<Option[Try[A]]>.toPot`.
```

Comment:
I wonder if `pure` would be a good alias for `ready` too

Reply:
If I'm not mistaken, `pure` should work (provided by `cats`) if you have the implicit `Applicative[Pot]` in scope (`import crystal.data.implicits._`).